### DISCUSSION TITLE
база

### DISCUSSION THREAD

**Discussion by voidlynx** [fdl-mc/freedomland#10] [General]
<a:arslee:762983440060252190>

**Comment by voidlynx** [comment]
не работает прикол
![image](https://user-images.githubusercontent.com/59738161/190473624-424e23da-4657-4f34-b84d-9552c63ac430.png)

**Comment by voidlynx** [comment]
арсли
 как удалить тред

**Reply by arslee07** [reply]
зачем удалять

**Reply by voidlynx** [reply]
я просто насрал немного, а кнопки "удалить" не вижу 
![basedge](https://user-images.githubusercontent.com/59738161/190966123-0fea3ffe-9dc1-4273-b48f-8472fa3b4510.png)
